### ISSUE TITLE
Dropout value in QuartzNet 15*5

### ISSUE THREAD

**Issue by MelikaBahmanabadi** [NVIDIA/NeMo#3021]
Hi
I fine-tuned the QuartzNet 15*5 model with 235 hours dataset of Persian Mozilla Common Voice. After that I restored the ckpt file and fine-tuned it with 25 hours of another persian dataset. Now Totally I have 50 epochs. From 18th epoch the loss start to increase. While before the 18th epoch the WER and loss both were decreased. So I deiced to change the dropout vale but in config file of QuartzNet 15*5 we have more than one dropout variable. Which of them  should I change to get the best result?

**Environment overview**

- Environment location: Google Colab
- 
- Method of NeMo install: !pip install nemo_toolkit[asr]
- 
- NeMo version: r1.2.0
- 
- Learning Rate: 2e-3

**Environment details**

- OS version : "Ubuntu20.04.3 LTS"
- PyTorch version : "1.9.1"

**Comment by titu1994** [comment]
You will have to change the dropout value of all but the last layer for QuartzNet
